### ISSUE TITLE
Scripting language - arrays - isIndex() function

### ISSUE THREAD

**Issue by rolandomunoz** [praat/praat#1360]
OS: Ubuntu 18.04
Praat version: 6.1.16

Arrays are useful for computing some tasks. It would be great if we could have a checking function to verify if an key value is already stored in an array. Something like:
```
tier["word"] = 4
if isIndex(tier, "word")
  # pass
endif
```

Not sure is this is a good idea or not.

Thank you

**Comment by Syuparn** [comment]
(I'm not a commiter of Praat though)
`variableExists` can be used to check whether it is defined.

```praat
tier["word"] = 4
if variableExists("tier[""word""]")
  appendInfoLine: "yes"
endif

**Comment by rolandomunoz** [comment]
Thank you. Yes, it works. Although the syntax is a bit cumbersome :(

**Comment by Syuparn** [comment]
I agree with you. `isIndex()` is more straightforward and easy to understand.
